### PR TITLE
Add preliminary labels to queries [VS-381]

### PR DESCRIPTION
**Pull request by rsasch**
Label names are standard from https://docs.google.com/document/d/1eyPrpCSCMyf1CaBkc-aDi39xV7Bq9hthh07ludpgPNw

branch adds labels to:
- `bg query` calls in WDL command blocks
- queries run within python scripts called by WDLs
- datasets created with Utils.BuildGATKJarAndCreateDataset

Quickstart integration run here: https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/15442e08-92af-48bc-ab96-2bdd4f39a801

### CODE REVIEW

File: scripts/variantstore/wdl/GvsAssignIds.wdl [hunk 1]
```diff
@@ -92,6 +92,8 @@ task AssignIds {
 
   Int samples_per_table = 4000
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label mangedby:assignids"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsAssignIds.wdl:96]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:assignids"
```

File: scripts/variantstore/wdl/GvsCreateAltAllele.wdl [hunk 2]
```diff
@@ -66,6 +66,8 @@ task GetVetTableNames {
   }
 
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label mangedby:createaltallele"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateAltAllele.wdl:70]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:createaltallele"
```

File: scripts/variantstore/wdl/GvsCreateAltAllele.wdl [hunk 3]
```diff
@@ -109,6 +111,8 @@ task CreateAltAlleleTable {
...
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateAltAllele.wdl:115]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:createaltallele"
```

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 4]
```diff
@@ -224,6 +224,8 @@ task ValidateFilterSetName {
   }
 
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label mangedby:extractcallset"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:228]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:extractcallset"
```

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 5]
```diff
@@ -472,6 +474,8 @@ task GenerateSampleListFile {
...
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:478]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:extractcallset"
```

File: scripts/variantstore/wdl/GvsImportGenomes.wdl [hunk 6]
```diff
@@ -250,6 +250,8 @@ task SetIsLoadedColumn {
   }
 
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label mangedby:importgenomes"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsImportGenomes.wdl:254]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:importgenomes"
```

File: scripts/variantstore/wdl/GvsImportGenomes.wdl [hunk 7]
```diff
@@ -296,6 +298,8 @@ task GetUningestedSampleIds {
   Int samples_per_table = 4000
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
   Int num_samples = length(external_sample_names)
+  # add labels for DSP Cloud Cost Control Labeling and Reporting
+  String bq_labels = "--label service:gvs --label team:variants --label mangedby:importgenomes"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsImportGenomes.wdl:302]:
```suggestion
  String bq_labels = "--label service:gvs --label team:variants --label managedby:importgenomes"
```

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py [hunk 8]
```diff
@@ -246,6 +246,9 @@ def make_extract_table(control_samples,
         if not (bool(re.match(r"[a-z0-9_-]+$", key)) & bool(re.match(r"[a-z0-9_-]+$", value))):
           raise ValueError(f"label key or value did not pass validation--format should be 'key1=val1, key2=val2'")
 
+    # add labels for DSP Cloud Cost Control Labeling and Reporting
+    query_labels.update({'service':'gvs','team':'variants','mangedby':'preparerangescallset'})
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:250]:
```suggestion
    query_labels.update({'service':'gvs','team':'variants','managedby':'preparerangescallset'})
```

File: scripts/variantstore/wdl/extract/populate_alt_allele_table.py [hunk 9]
```diff
@@ -13,7 +13,8 @@
 
 def populate_alt_allele_table(query_project, vet_table_name, fq_dataset, sa_key_path):
     global client
-    default_config = QueryJobConfig(priority="INTERACTIVE", use_query_cache=True)
+    # add labels for DSP Cloud Cost Control Labeling and Reporting to default_config
+    default_config = QueryJobConfig(priority="INTERACTIVE", use_query_cache=True, labels={'service':'gvs','team':'variants','mangedby':'createaltallele'})
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/populate_alt_allele_table.py:17]:
```suggestion
    default_config = QueryJobConfig(priority="INTERACTIVE", use_query_cache=True, labels={'service':'gvs','team':'variants','managedby':'createaltallele'})
```

**Review by mcovarr — COMMENTED** [review]
I'm assuming that's a mangy typo, otherwise please ignore... 

OOC any reason not to use underscores in the core workflow names to improve readability?

**Comment by rsasch** [comment]
> I'm assuming that's a mangy typo, otherwise please ignore...

Indeed — thanks for catching that!
 
> OOC any reason not to use underscores in the core workflow names to improve readability?

Nope, done

**Review by mcovarr — APPROVED** [review]
LGTM 👍

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py [hunk 10]
```diff
@@ -246,6 +246,9 @@ def make_extract_table(control_samples,
         if not (bool(re.match(r"[a-z0-9_-]+$", key)) & bool(re.match(r"[a-z0-9_-]+$", value))):
           raise ValueError(f"label key or value did not pass validation--format should be 'key1=val1, key2=val2'")
 
+    # add labels for DSP Cloud Cost Control Labeling and Reporting
+    query_labels.update({'service':'gvs','team':'variants','managedby':'prepare_ranges_callset'})
+
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:251]: dont forget to make a new docker image to use these changes

Comment (rsasch) [reply scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:251]: good catch — thanks!

**Pull request merged**